GOVERNANCE.md

```diff
@@ -103,29 +180,93 @@ Steward is an additional privilege bestowed to 1 (or more) Contributors. The rol
+1. **If the vote fails:** the project steward is responsible for informing the nominee with constructive, actionable feedback. (Note: this is not required if the nomination was made in the #core channel, or if the nominee was otherwise not made aware of their nomination).
+
+## Voting: Governance Change
+
+A vote is initiated once a pull request to the GOVERNANCE.md file is submitted by a Core Maintainer.
+
+If the pull request submitter is not a Core Maintainer, the PR can be closed by any Maintainer without a vote. However, any Core Maintainer may request a vote on that PR, in which case a vote is initiated.
+
+**Who can vote:** Core Maintainers (L3 and above). All Maintainers are encouraged to discuss and voice their opinion in the pull request discussion. Core Maintainers should take the opinions of others into consideration when voting.
+
+1. The pull request discussion thread is used to discuss the governance change.
+2. The normal 3 day voting & discussion window begins with either the PR creation or the removal of `WIP:` from the PR title if the PR was created as a draft.
+3. Voting can be done in the pull request via a review of either **Approve (For)** or **Change Requested (Against)**.
+4. The vote must receive a simple majority (50%+) to pass.
+5. **If the vote passes:** the PR is merged and the changes take effect immediately.
+6. **If the vote fails:** the PR is closed and no change occurs.
+
+
+## Voting: RFC Proposals
+
+Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.
+
+**Who can vote:** All Maintainers (L2 and above).
+
+1. Anyone can submit an RFC to suggest changes to Astro.
+2. A trivial change can be discussed and approved entirely within the RFC GitHub issue, as long as there are no objections from Maintainers or Core Maintainers. This is not considered a formal vote.
+3. A non-trivial, significant change should be discussed within the RFC GitHub issue and approved during an RFC meeting call.
+4. During an RFC meeting, the moderator will attempt to achieve consensus on the RFC proposal.
+5. **If consensus is ever reached:** the RFC is approved.
+6. Otherwise, Maintainers should attempt to reach consensus over the course of the next week, before bringing the RFC back to an RFC meeting. All reasonable attempts to address concerns should be exhausted.
+7. **If consensus still cannot be reached:** The project Steward may initiate the first fallback mechanism by limiting the vote to Core Maintainers.
+8. **If consensus still cannot be reached:** The project Steward may initiate a final fallback vote of Core Maintainers, of which an overwhelming majority (80%+) is required to pass.
```

Comment:
7 and 8 are both votes by core maintainers, did you mean for 7 to be just maintainers and 8 to be core maintainers? If not, what is the difference between 7 and 8?

Reply:
The idea is:
Attempt 1: Reach consensus among all Maintainers (L2+)
Attempt 2: if consensus cannot be reached reasonably,  fallback to only Core Maintainers (L3)
Attempt 3: if consensus still cannot be reached reasonably,  attempt an overwhelming majority vote of Core Maintainers
End: If consensus still cannot be reached reasonably, RFC fails.

Reply:
That's meant to be clearly stated in the list above, but open to any edits that would make this more clear.

Reply:
Could 5 and 6 be combined? The general idea being that ideally consensus should happen within the meeting, but also may happen throughout the week in GitHub/Discord.

Also maybe putting a boundary on 7 may help: “If consensus cannot be reached after a week, …“

Reply:
cleaned up! I didn't want to put an exact time limit into the governance doc unless required, but I'll point out that the word "should" is used intentionally here to give the project Steward room to use their best judgement when initiating a fallback.